src/lib/mongoBackend/mongoQueryTypes.cpp

```diff
@@ -149,6 +149,126 @@ static long long countEntities(const std::string& tenant, const std::vector<std:
 
 /* ****************************************************************************
 *
+* mongoEntityTypesValues -
+*
+* "Simplified" version of mongoEntityTypes(), using a simpler aggregation command
+* and the processing logic afterwards. Note hat apiVersion is not included in this
+* operation as it can be only used in NGSIv2
+*/
+HttpStatusCode mongoEntityTypesValues
+(
+  EntityTypeVectorResponse*            responseP,
+  const std::string&                   tenant,
+  const std::vector<std::string>&      servicePathV,
+  std::map<std::string, std::string>&  uriParams,
+  unsigned int*                        totalTypesP
+)
+{
+  unsigned int   offset         = atoi(uriParams[URI_PARAM_PAGINATION_OFFSET].c_str());
+  unsigned int   limit          = atoi(uriParams[URI_PARAM_PAGINATION_LIMIT].c_str());
```

Comment:
It's not very import and and it has not much to do really with this PR.
Some day we should look into it though.
Also, why have the `atoi(uriParams[URI_PARAM_PAGINATION_OFFSET].c_str()` in more than one place ... (should be str2double or something like that).
Creating an issue.

NTC